snippets/csharp/VS_Snippets_Wpf/CustomControlNumericUpDown/CSharp/CustomControlLibrary/NumericUpDown.cs

```diff
@@ -10,21 +10,10 @@
 
 namespace CustomControlLibrary
 {
-    //<SnippetStaticCtorOfCustomClassCommonTasks>
-    public class NumericUpDown : Control
+    public partial class NumericUpDown : Control
     {
-        static NumericUpDown()
-        {
-            InitializeCommands();
-
-            // Listen to MouseLeftButtonDown event to determine if slide should move focus to itself
-            EventManager.RegisterClassHandler(typeof(NumericUpDown), 
-                Mouse.MouseDownEvent, new MouseButtonEventHandler(NumericUpDown.OnMouseLeftButtonDown), true);
-
-            DefaultStyleKeyProperty.OverrideMetadata(typeof(NumericUpDown), new FrameworkPropertyMetadata(typeof(NumericUpDown)));
-        }
-        //</SnippetStaticCtorOfCustomClassCommonTasks>
-
+// <staticctorofcustomclasscommontasks>
```

Comment:
do you need to keep this or you just don't want to break the build?

Reply:
I don't want to break the build. Once we've merged, I'll remove it.